### PR TITLE
adds final destination deathmatch map

### PR DESCRIPTION
**Pull request by sylvia-from-fulp-station**
## About The Pull Request
https://github.com/tgstation/tgstation/pull/82929 
I spent hours working on this; mapping, setting classes, testing, balancing. It feels like shit to see pr closed. This is the same map but without the removal of mech madness in the pr

Photoshop_NFdOZPhR1L
nuke op has a bulldog instead

## Why It's Good For The Game
its a deathmatch map; there are many like it but this one is mine. The classes represent a variety of characters you'd find aboard space station 13, albeit cartoonishly powerful and with more gear than you could reasonably use. Have some honourable showdowns at peak performance or a chaotic loot fest; no gear is restricted. 

## Changelog
:cl:
add: final destination deathmatch map, loadouts
fix: indestructible light tile emits light
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
needs items strewn around and a special deathmatch modifier for 'no items'

also another that is 'host loadout only'

that way we can finally have: final destination, no items, tider only

**Comment by sylvia-from-fulp-station** [comment]
> needs items strewn around and a special deathmatch modifier for 'no items'
> 
> also another that is 'host loadout only'
> 
> that way we can finally have: final destination, no items, tider only

beyond the scope here but im all for it; this map was made with first version of deathmatch in mind, before modifiers.
design wise; i've considered adding items since the start but thought it goes against the attempt at a "competitive" map, but with a switch it's closer to the inspiration + more fun.

**Comment by MrMelbert** [comment]
i recently watched a 30 minute video detailing how problematic FD is as a stage design so are we sure we should port this

**Comment by sylvia-from-fulp-station** [comment]
> i recently watched a 30 minute video detailing how problematic FD is as a stage design so are we sure we should port this

Do any of the points apply to this map? It's themed like smash bros and designed to be well, square; but its not just a flat platform; the girders and central chasms were put in to make it more dynamic; but they remain symmetrical as to present the "rules of engagement" rather than chasms in other maps which attempt to trick you into falling to your death at the start of the match. You can absolutely have a serious, FD, no items, style fight here, or you can have 8 people pick a silly no armour grenadier belt loadout. It's player dependent.

I want to port it because i made it and people enjoyed it. I have put a lot of thought unironically into making it a "competitive" ss13 map, whatever the fuck that means. 

tldr: It's not final destination from smash bros, its final destination for ss13 deathmatch.

**Comment by MrMelbert** [comment]
> tldr: It's not final destination from smash bros, its final destination for ss13 deathmatch.

<Details>

<Summary> </Summary>

thatsthejoke

</Details>

**Comment by sylvia-from-fulp-station** [comment]
> cough
im gonna cry now

**Comment by munchyi** [comment]
nice

**Comment by SimplySmiley123** [comment]
Loved this DM map, really fun to mess around on it.

**Comment by carlarctg** [comment]
needs an incredibly distracting and nauseating background that ruins the competitive aspect of it
